integr8ly/cloud-resource-operator
# [WIP] first draft adding sts tag to redis/postgres resources in aws

## Overview

Jira: https://issues.redhat.com/browse/MGDAPI-1726

## Verification

WIP pr, no need to verify yet.


- Clone this branch
- Run `make cluster/prepare`
- Run `make run`
- Ensure...

## Checklist
